pre-docs/bolt_installing_modules.md

```diff
@@ -0,0 +1,26 @@
+
+# Installing modules with Bolt
+
+Bolt uses a Puppetfile to specify which modules to install. The Puppetfile
+should be in the Boltdir, which can either be a project-specific directory
+named `Boltdir` at the root of your project, or a global directory in
+`$HOME/.puppetlabs/bolt`.
+
+The Puppetfile contains a list of modules and versions to install. If the
+modules have dependencies on other modules, those dependencies will need to be
+listed in the Puppetfile as well.
+
+By default, modules will be downloaded from the Puppet Forge. Modules can be
+installed from git repositories instead by setting the `git` and `ref`
+properties.
+
+```
+mod 'puppetlabs/package', '0.2.0'
+mod 'puppetlabs/service', '0.3.1'
+mod 'puppetlabs/puppetlabs-facter_task', git: 'git@github.com:puppetlabs/puppetlabs-facter_task.git', ref: 'master'
```

Comment:
Do we want an example of a local module or should we tell users to put local modules in Boltdir/site? We don't support modulepaths relative to boltdir yet do we?

Reply:
I'll add a note about that. I don't think we support relative modulepaths, but I'm also not 100% sure what would happen if you tried.